docs/packs/packs-format.md

```diff
@@ -59,6 +59,7 @@ The following fields are populated in the pack metadata:
 | dependencies | Dictionary | (Optional) An object that describes the content packs that the pack is dependant on. Should be kept empty on pack creation, as it is calculated by Cortex XSOAR content infrastructure. |
 | displayedImages | List | (Optional) Images to be displayed in Cortex XSOAR marketplace. Should be kept empty on pack creation, as it is calculated by Cortex XSOAR content infrastructure. |
 | githubUser | List | (Optional) List of Github usernames to receive notification in the PR in case pack files were modified. |
+| devEmail | String | (Optional) Email of developer, in order to receive notification in case contributed pack files were modified. |
```

Comment:
Should be list type
```suggestion
| devEmail | List | (Optional) List of emails, to receive notification in case contributed pack files were modified. |
```